.travis.yml

```diff
@@ -70,7 +70,7 @@ script:
   - if [[ "${RUBOCOP}" == "yes" ]]; then rubocop; fi
 
   # Test ruby gem
-  - gem install gem-path --no-ri --no-rdoc
+  - gem install gem-path --no-document 
```

Comment:
In latest version of Ruby (> 2.2) , we need to use `--no-document`, because `--no-ri --no-rdoc` is deprecated.